modules/encoders/x86/unicode_mixed.rb

```diff
@@ -23,6 +23,11 @@ def initialize
         {
           'BlockSize' => 1,
         })
+    register_options(
+      [
+        OptString.new('BufferRegister',[true, "The register that points to the encoded payload", "ECX"])
```

Comment:
I didn't want to nitpick style, but please use consistent quoting and whitespace next time.

```suggestion
        OptString.new('BufferRegister', [true, 'The register that points to the encoded payload', 'ECX'])
```

Also, there is a typo in the library option's description that should be fixed:

```
wvu@kharak:/rapid7/metasploit-framework:master$ git grep BufferRegister lib
lib/msf/core/encoder/alphanum.rb:        OptString.new('BufferRegister', [ false, "The register that pointers to the encoded payload" ]),
wvu@kharak:/rapid7/metasploit-framework:master$
```

Option descriptions should match for a given name, which is important when overriding with `register_default_options` instead of `DefaultOptions`, since any discrepancies won't offer a good user experience.

Thank you for your patch. :)

cc @gwillcox-r7

Reply:
@wvu-r7 Do you want me to raise a separate PR to fix this?

Reply:
It's not a big deal, but now that I look at it, it is kind of bothering me. The descriptions should at least match, since we're reregistering the option to override it instead of using the designated `DefaultOptions` feature.

Reply:
Ok let me open a PR to fix this up, as I think the library should:

1. Not be using double quotes for a literal string.
2. Should have `that pointers` changed to `that points` to fit in with other modules.

Reply:
Yeah, and `,[` should be `, [`, at least.

Reply:
I'm personally fine with double quotes for literal strings, but we have to be consistent about it. Single quotes generally have fewer side effects for literals.

In this case, `'BufferRegister'` was already using single quotes, so it makes sense to use them for the rest of the line. That is all.

Reply:
Other than the option descriptions not matching, all these are nitpicks.

Reply:
It is also partly the reason I suggested https://github.com/rapid7/metasploit-framework/pull/13375#discussion_r418710835 and `DefaultOptions`, but we didn't want to rework the library - that's fine.

Reply:
Note that #9000 is also an option. I just can never remember the invocation.

Reply:
@wvu-r7 Made #13380 to fix these issues.